src/app/main/service/habit/habit.service.ts

```diff
@@ -22,8 +23,10 @@ export class HabitService {
   private backEnd = environment.backendLink;
   private httpOptions = {
     headers: new HttpHeaders({
+      'Content-Type': 'application/json',
       Authorization: 'my-auth-token'
-    })
+    }),
+    withCredentials: true
```

Comment:
Also need to remove - `'Content-Type': 'application/json'`, and `withCredentials: true`